README.md

```diff
@@ -0,0 +1,134 @@
+var ethAccount: {address: SDK.ledger.Address, signer: ethers.Signer};
+// Create client object.
+let Erdstall = SDK.NewClient(
+	ethAccount.address,
+	ethAccount.signer,
+	new URL("https://operator.erdstall.dev"));
+// Set up any event listeners needed by the application.
+Erdstall.once("open", () => { myDApp.run(Erdstall); });
+// Establish the client's connection.
+Erdstall.initialize();
+```
+
+**Setting up events**&emsp;<!--
+-->After creation, the Erdstall client is now only partially initialized.
+Now is the time to set up any `SDK.ledger.ErdstallEvent` and `SDK.enclave.EnclaveEvent` handlers using `Erdstall.on()` and `Erdstall.once()`.
+The following `EnclaveEvent` events exist (emitted event types are written in parentheses):
+
+* **`"open"`**:
+		triggered when the connection to Erdstall is established.
+	This event is triggered again if the Erdstall connection is lost and automatically re-established.
+	This is the most important event and must be set during setup, or it might get lost.
+	Use this event to launch your Erdstall-related logic.
+* **`"receipt"(SDK.TxReceipt)`**:
+		triggered when a receipt is received from a transaction subscription.
+	This does not include transaction receipts directly received as a response when issuing transactions.
+	A transaction receipt contains the transaction, an affected party, and the party's new balance.
+* `"close"`: triggered when the connection to Erdstall is terminated due to any reason.
+* `"error"`: triggered when an error occurred with the Erdstall connection.
+* `"proof"(SDK.BalanceProof)`:
+		triggered when a balance proof is received from a proof subscription.
+	These balance proofs are used for exiting the system in case the operator ceases to function.
+* `"exitproof"(SDK.BalanceProof)`:
+		triggered when a balance proof intended for exiting Erdstall on-chain is received from a proof subscription.
+	This happens after a call to `Erdstall.exit()` or during `Erdstall.leave()`.
+
+The following `ErdstallEvent` smart contract events exist:
+
+* **`"Deposited"`**:
+		you deposited money into Erdstall.
+* **`"Withdrawn"`**:
+		you withdrew funds from Erdstall.
+* `"Frozen"`:
+		an unrecoverable error occurred and the Erdstall system is now frozen forever, and funds must be withdrawn using the latest "sealed" balance proof.
+* `"Challenged"`:
+		someone challenged the Erdstall operator to prove it is still alive.
+* `"ChallengeResponded"`:
+		the Erdstall operator successfully proved it still operates.
+* `"TokenTypeRegistered"`:
+		a new token type (such as ETH, ERC20, ERC721, etc.) has been registered with Erdstall on-chain.
+* `"TokenRegistered"`:
+		a specific token contract has been registered on-chain.
+
+
+**Finishing up**&emsp;<!--
+-->After all events have been set up, call `Erdstall.initialize()` to establish the client's connection to the Erdstall ledger.
+You're now good to go!
+The `open` event will be triggered and your logic will launch.
+
+## Using the Erdstall client
```

Comment:
It would be great to add a reference to a minimal working example.

Reply:
created new issue #13